### ISSUE TITLE
Missing FQDN/hostname option for windows

### ISSUE THREAD

**Issue by patryk4815** [shirou/gopsutil#1572]
Missing FQDN option for windows:

Example implementation:
https://github.com/elastic/go-sysinfo/blob/main/providers/windows/host_windows.go#L88C44-L88C81

**Comment by shirou** [comment]
Indeed, this might be a good feature to have. And it looks like it could be implemented on all platforms. Thanks for the suggestion.
